client/my-sites/hosting/staging-site-card/card-content/staging-sync-card.tsx

```diff
@@ -429,8 +434,8 @@ export const SiteSyncCard = ( {
 						className="staging-site-sync-card__radio"
 						label={
 							type === 'production'
-								? translate( 'Pull from staging' )
-								: translate( 'Pull from production' )
+								? translate( 'Pull staging into production' )
```

Comment:
I'm not convinced it's a good direction. If we include both source and target in the message, wouldn't it make more sense to say "Synchronize staging into production"? 

Or another approach. If the label says "Choose synchronization direction", would it make sense to provide "From staging to production" and "From production to staging" options?

Reply:
We aren't synchronizing, though. We're pushing and pulling data.